homeassistant/components/daikin/switch.py

```diff
@@ -119,3 +124,41 @@ async def async_turn_on(self, **kwargs: Any) -> None:
     async def async_turn_off(self, **kwargs: Any) -> None:
         """Turn the zone off."""
         await self._api.device.set_streamer("off")
+
+
+class DaikinToggleSwitch(SwitchEntity):
+    """Switch state."""
+
+    _attr_icon = TOGGLE_ICON
+    _attr_has_entity_name = True
+
+    def __init__(self, daikin_api: DaikinApi) -> None:
+        """Initialize switch."""
+        self._api = daikin_api
```

Comment:
To be more consistent with other part of the code in Daikin:

```suggestion
    def __init__(self, api: DaikinApi) -> None:
        """Initialize switch."""
        self._api = api
        self._attr_device_info = api.device_info
        self._attr_unique_id = f"{self._api.device.mac}-toggle"
```